### ISSUE TITLE
Allow/disallow using callback function

### ISSUE THREAD

**Issue by larvanitis** [mistakster/postcss-unwrap-at-media#2]
It would be useful to have the option to provide a custom function instead of just strings.

eg.
```js
function disallow(atRule) {
  return !/print/.test(atRule.params);
}

postcss([ require('postcss-unwrap-at-media')((atRule) => {
  return !/print/.test(atRule.params);
}) ])
```

**Comment by mistakster** [comment]
Hi Leonidas! Your suggestion makes sence. I will implement it.

**Comment by mistakster** [comment]
Fixed in version `2.1.1`. See [Configuration](https://github.com/mistakster/postcss-unwrap-at-media#configuration) section in README.
